parcels/examples/example_dask_chunk_OCMs.py

```diff
@@ -226,9 +226,8 @@ def test_pop(mode, chunk_mode):
     if chunk_mode is False:
         assert (len(field_set.U.grid.load_chunk) == 1)
     elif chunk_mode == 'auto':
-        assert (len(field_set.U.grid.load_chunk) != 1)
+        assert (len(field_set.U.grid.load_chunk) == 1)
     elif chunk_mode == 'specific':
-        print(field_set.U.grid.chunk_info)
```

Comment:
yes, that can go - but for analysis, it would be nice to know the output for the above case.

Reply:
The result of this `print(field_set.U.grid.chunk_info)` statement (placed in the `elif chunk_mode == 'auto'` clause) is now `[3, 1, 1, 1, 20, 60, 60]`, while it is `[3, 2, 1, 1, 20, 1, 60, 60]` in the current master. Note that the second entry has changed from `2` to `1` due to this PR. Not sure why..

Reply:
yes, I see the change - I don't see exactly why that's the case unless the data layout has changed ...

Reply:
overall, this here should not occur (i.e. the `load_chunk` field should not have a length of 1) unless (a) the data is in a flattened array or (b) the test data is too small for further chunking. In the latter case, we can check if lowering the chunk buffer size at the start of  that method changes the outcome of this if-check statement.

Reply:
Well, the only unit test I had to change is the `chunk_mode='auto'` one. The `specific` still works. I think that indeed, because we have now one depth level less, `auto` now defaults to only one chunk